local/blockchain.go

```diff
@@ -458,7 +458,7 @@ func (ln *localNetwork) restartNodesWithWhitelistedSubnets(
 		delete(nodeConfig.Flags, config.WhitelistedSubnetsKey)
 
 		ln.log.Info("removing and adding back the node for whitelisted subnets", zap.String("node-name", nodeName))
-		if err := ln.restartNode(ctx, nodeName, "", "", nil, nil, nil); err != nil {
+		if err := ln.restartNode(ctx, nodeName, "", "", "", nil, nil, nil); err != nil {
```

Comment:
Note, not to be addressed in this PR but possibly create a ticket: These naked parameters are close to impossible to reason about what each means. I think we should not do this?

Reply:
https://github.com/ava-labs/avalanche-network-runner/issues/390